astrobee/launch/sim.launch

```diff
@@ -28,13 +28,14 @@
   <arg name="nodes" default=""/>                <!-- Launch specific nodes   -->
   <arg name="extra" default=""/>                <!-- Inject additional node  -->
   <arg name="debug" default=""/>                <!-- Debug node group        -->
-  <arg name="physics" default="ode"/>           <!-- Physics engine          -->
+  <!-- Physics engine options: ode (gazebo default), bullet, simbody, dart   -->
+  <arg name="physics" default="dart"/>
```

Comment:
Do we need to change this?

I vaguely recall you found that `dart` worked better than `ode` in some corner case, and your testing was partly motivated because you thought `dart` could aid Rika's research, but then when I talked to Rika she thought it wasn't needed and suggested that she had other modeling tricks to make the dynamics in her scenarios better-conditioned.

I guess the bottom line is this seems like something that could break simulation scenarios for some of our users in unpredictable ways, and I'm not clear who benefits. So maybe it's not a good idea to change right now without more discussion. It's already easily available for users who want to switch, correct?